tools/cosmovisor/README.md

```diff
@@ -6,21 +6,21 @@ sidebar_position: 1
 
 `cosmovisor` is a small process manager for Cosmos SDK application binaries that monitors the governance module for incoming chain upgrade proposals. If it sees a proposal that gets approved, `cosmovisor` can automatically download the new binary, stop the current binary, switch from the old binary to the new one, and finally restart the node with the new binary.
 
-* [Cosmovisor](#cosmovisor)
-    * [Design](#design)
-    * [Contributing](#contributing)
-    * [Setup](#setup)
-        * [Installation](#installation)
-        * [Command Line Arguments And Environment Variables](#command-line-arguments-and-environment-variables)
-        * [Folder Layout](#folder-layout)
-    * [Usage](#usage)
-        * [Initialization](#initialization)
-        * [Detecting Upgrades](#detecting-upgrades)
-        * [Auto-Download](#auto-download)
-    * [Example: SimApp Upgrade](#example-simapp-upgrade)
-        * [Chain Setup](#chain-setup)
-            * [Prepare Cosmovisor and Start the Chain](#prepare-cosmovisor-and-start-the-chain)
-            * [Update App](#update-app)
+- [Cosmovisor](#cosmovisor)
```

Comment:
Can you revert this?

Reply:
This should still be reverted.

Reply:
I think I did this - lmk. It looks good in my commit but the diff doesn't seem to be showing it